templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,11 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 35**
+
+* The peer_add and peer_remove subscription events now have plural
+  versions of `user_ids` and `stream_ids`.
```

Comment:
We should also add a `**Changes**` entry in the `peer_add` / `peer_remove` event definitions in `zulip.yaml` (just mentioning what the old format was in text).